front/src/modules/trainschedule/components/Timetable/types.ts

```diff
@@ -43,3 +43,8 @@ export type InvalidReason =
   | Extract<SimulationSummaryResult['status'], 'pathfinding_failure' | 'simulation_failed'>
   | PathfindingNotFound['error_type']
   | PathfindingInputError['error_type'];
+
+export type TheoreticalMarginsRecord = Record<
+  string,
+  { theoreticalMargin: string; isBoundary: boolean }
+>;
```

Comment:
You can move this type in the types of `modules/timesStops` since it used only there

Reply:
Oops, I confused timetable and timesstops, thanks!